hie.yaml

```diff
@@ -0,0 +1,106 @@
+cradle:
+  cabal:
+    - path: "byron/chain/executable-spec/src"
+      component: "lib:byron-spec-chain"
+
+    - path: "byron/chain/executable-spec/test"
+      component: "byron-spec-chain:test:chain-rules-test"
+
+    - path: "byron/crypto/src"
+      component: "lib:cardano-crypto-wrapper"
+
+    - path: "byron/crypto/test"
+      component: "cardano-crypto-wrapper:test:test"
+
+    - path: "byron/crypto/test/./"
+      component: "lib:cardano-crypto-test"
```

Comment:
Don't these paths conflict? I suppose you're using a generator for this which doesn't handle everything correctly.